docs/GettingStartedDocs/buildandsign.md

```diff
@@ -97,6 +97,18 @@ NumHeapPages=1024
 NumStackPages=1024
 NumTCS=1
 ```
+Additionally, a developer can specify additional Key Sharing and Separation (KSS) identity properties
+for use on the platforms that support it:
+- **FamilyID**: The product family identity (ISVFAMILYID) a developer can specify to group
+different enclaves under a common identity. For example, an identifier for the application suite
+which includes several enclave apps.
+- **ExtendedProductID**: The extended product identity (ISVEXTPRODID) value that a developer
+can use as 128-bit globally unique identifier for the enclave where the 16-bit ProductID proves too restrictive.
+For example, in kss_valid.conf:
+```
+FamilyID=47183823-2574-4bfd-b411-99ed177d3e43
+ExtendedProductID=47183823-2574-4bfd-b411-99ed177d3e43
+```
```

Comment:
The explanation of the updated `OE_SET_ENCLAVE_SGX` macro (or the suggested additional `OE_SET_ENCLAVE_SGX_KSS` macro) need to be updated in this doc.

Reply:
Updated in #3700

Reply:
Updated in #3700.